package.json

```diff
@@ -27,7 +29,10 @@
   },
   "homepage": "https://github.com/pmadruga/react-native-clean-project#readme",
   "devDependencies": {
-    "eslint": "5.2.0",
+    "eslint": "^5.16.0",
     "jest": "^24.8.0"
+  },
+  "rnpm": {
+    "plugin": "./source/plugin.js"
```

Comment:
I guess it's normal boilerplate to have the './source/plugin' reference both here and in the react-native.config.js. Should they always match?

Reply:
Yes - one is for react-native-community/cli v1 (the rnpm config) and one is for v2 (where they migrate to react-native.config.js. I wanted it to work now and in the pending release so I included the right hooks for both. I don't believe we can do comments in JSON or I'd leave a comment to remove the rnpm implementation later (once CLI v1 is long gone) but that would be the idea